### ISSUE TITLE
2.9.0 release not compatible with older mocha version

### ISSUE THREAD

**Issue by dRacz3** [hbenl/vscode-mocha-test-adapter#139]
Hi,

VSCode installed automatically the newest 2.9.0. version, and the extension breaked immediately. I'm getting the following error:
```javascript
TypeError: mocha.parallelMode is not a function
    at /Users/username/.vscode/extensions/hbenl.vscode-mocha-test-adapter-2.9.0/out/worker/bundle.js:5722:19
    at Generator.next (<anonymous>)
    at fulfilled (/Users/username/.vscode/extensions/hbenl.vscode-mocha-test-adapter-2.9.0/out/worker/bundle.js:38:58)
```

My vscode settings (not sure if it helps, but here it is):
```json
{
    "mochaExplorer.files": "test/**/*.js",
    "mochaExplorer.require": "babel-core/register",
}
```

I'm using macOS Catalina (Version 10.15.7), node v12.19.0 (npm v6.14.8), mocha version: 7.2.0,

I have rolled back to the previous extension version (2.8.1), and everything was working again.

**Comment by dRacz3** [comment]
Update:

I have also tried to update the mocha version to 8.2.1, which seemed to solve the issue, so the extension seems to be working, but not backward compatible with older mocha versions such as my current (7.2.0)

Not sure if the compatibility loss is intended or not. If it is intended, it would be a good to have a notice on the extension page.

**Comment by hbenl** [comment]
Sorry about that, the compatibility loss was definitely not intended. I fixed it in version 2.9.1.

**Comment by dRacz3** [comment]
Thank you for the quick action, I can confirm that it is working now as intended! :) Good job!
